src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -616,3 +617,37 @@
                                  (icons/icon {} (if (:collapsed? @state) :expand :collapse)))
           body]
          body)))})
+
+(react/defc QueueStatus
+  {:render
+   (fn [{:keys [state this]}]
+     (let [{:keys [queue-status queue-error]} @state
+           {:keys [queue-time queue-position queued active]} queue-status]
+       [:div {:style {:marginBottom "0.5em"}}
+        (cond
+          queue-error (style/create-server-error-message queue-error)
```

Comment:
I tested this by changing the url of `endpoints/submissions-queue-status` to be a 404. 

When there is an error, the error is displayed without any indication of what the component is supposed to be. In my case, it meant the words "Not Found" were floating in the page ... but I wouldn't have been able to tell *what* was not found. Maybe prefix the error message with something like "Queue Status" or "Error retrieving queue status".

On the Monitor tab, the error message is centered on the page, while the successful queue status is left-aligned.

Finally - if the user does get an error, the "refresh" link is gone. The user has no way to retry the queue status without navigating away or refreshing the page.

Reply:
Yeah, I didn't test that component too thoroughly when I moved it... the "Not Found" happens on dev as well it looks like. Will try to come up with something nicer.

Reply:
Fixed the message to be informative and centered...

<img width="1495" alt="screen shot 2017-10-16 at 3 03 15 pm" src="https://user-images.githubusercontent.com/1087943/31629971-2c214d78-b283-11e7-86d0-732c02ede983.png">

...and added a refresh button to recover

![untitled](https://user-images.githubusercontent.com/1087943/31629905-fb0a06d0-b282-11e7-938e-3128ca6439c0.gif)